src/network/taraxa_capability.cpp

```diff
@@ -298,7 +298,7 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
       LOG(log_dg_dag_prp_) << "Received NewBlockPacket " << transactionsCount;
 
       std::vector<Transaction> newTransactions;
-      for (auto iTransaction = 1; iTransaction < transactionsCount + 1; iTransaction++) {
+      for (unsigned long iTransaction = 1; iTransaction < transactionsCount + 1; iTransaction++) {
```

Comment:
why not size_t ? + refactored auto transactionsCount from a few lines above ?

Reply:
good point, I will fix it